scripts/gen_lazy_tensor.py

```diff
@@ -112,7 +112,7 @@ def build_ir_node(self, func: NativeFunction, schema: LazyIrSchema) -> str:
       get_tensorlist="GetTensorList",
       get_tensor_or_wrap_number="bridge::GetXlaTensorOrCreateForWrappedNumber",
       try_get_tensor="bridge::TryGetXlaTensor",
-      metrics_counter='XLA_FN_COUNTER("xla::")',
+      metrics_counter='TORCH_LAZY_FN_COUNTER("xla::")',
```

Comment:
I think `TORCH_LAZY_FN_COUNTER` is default so maybe we don't need to pass it? I am not sure if `xla::` will also be there through.

Reply:
The default is "lazy::". We need to replace that with "xla::".